modules/data.atmosphere/R/met_temporal_downscale.Gaussian_ensemble.R

```diff
@@ -330,7 +330,7 @@ met_temporal_downscale.Gaussian_ensemble <- function(in.path, in.prefix, outfold
         if (precip[i] == 0) {
           p <- 2
         }
-        hdry[i] <- A * p * (1 - exp(-1 * bmlist[i] * ((temp_max[i] - temp_min[i])^C)))
+        hdry[i] <- A * p * (1 - exp(-1 * bmlist[i] * ((df$temp_max[i] - df$temp_min[i])^C)))
```

Comment:
How sure are you that `temp_max` and `temp_min` come from `df`? Since `df` starts out empty on line 112  I think we'd be able to see where they were defined if they're there... but this code is definitely confusing and I don't see where else they would have come from

Reply:
I'm going to revert this change and go back to ignoring the NOTE about `temp_max` and `temp_min`.

I went back through the history of this file and as far as I can tell `temp_max` and `temp_min` have *never* had any visible binding. Whether the code is working as intended or not, it's been this way for a long time and can wait for someone to investigate in more detail.

Reply:
Done in 691aa6b4f